targets.json

```diff
@@ -126,6 +126,16 @@
                 "firmware": "Unified_ESP32_900_TX",
                 "prior_target_name": "BETAFPV_900_TX"
             },
+            "nanov2": {
+                "product_name": "BETAFPV 900MHz Nano TX V2",
+                "lua_name": "BETAFPV 900Nano",
+                "layout_file": "BETAFPV 900 Nano V2.json",
+                "upload_methods": ["uart", "wifi"],
+                "min_version": "3.4.0",
```

Comment:
Any show stoppers preventing this being the v3.3.0, or earlier version?

Reply:
Nothing at all, so I changed them